requirements-dev.txt

```
@@ -0,0 +1,7 @@
+pytest
```

I think you can add this line and just run requirements-dev.txt for both
```suggestion
-r requirements.txt
pytest
```